src/rspq/rspq.c

```diff
@@ -489,9 +492,9 @@ static void rspq_get_current_ovl(rsp_queue_t *rspq, int *ovl_idx, const char **o
 /** @brief RSPQ crash handler. This shows RSPQ-specific info the in RSP crash screen. */
 static void rspq_crash_handler(rsp_snapshot_t *state)
 {
-    rsp_queue_t *rspq = (rsp_queue_t*)state->dmem;
+    rsp_queue_t *rspq = (rsp_queue_t*)(state->dmem + RSPQ_DATA_ADDRESS);
     uint32_t cur = rspq->rspq_dram_addr + state->gpr[28];
-    uint32_t dmem_buffer = RSPQ_DEBUG ? 0x140 : 0x100;
+    uint32_t dmem_buffer = RSPQ_DEBUG ? 0x1A0 : 0x100;
```

Comment:
OK I'll do that on unstable and port it to trunk. rspq on unstable had a refactoring that shuffled things around. I want to merge that first otherwise I always need to do changes twice.